website/docs/r/signalr_service.html.markdown

```diff
@@ -73,6 +82,18 @@ A `features` block supports the following:
 
 ---
 
+An `upstream_setting` block supports the following:
```

Comment:
i think think we can remove setting here.

This block seems to match things, so maybe
```suggestion
An `upstream_match` block supports the following:
```

or match upstream?

Reply:
Based off my reading of [Azure's documentation](https://docs.microsoft.com/en-us/azure/azure-signalr/concept-upstream) around this feature I'm not convinced either of our options are best. 

> Upstream is a preview feature that allows Azure SignalR Service to send messages and connection events to a set of endpoints in serverless mode. You can use upstream to invoke a hub method from clients in serverless mode and let endpoints get notified when client connections are connected or disconnected.

Based off my experience with SignalR in the past I'd imagine this is a "forwarding" mechanism for SignalR service events (clients connected, disconnected, message sent, etc.). 

I am thinking that `upstream` or `upstream_endpoint` would better represent the intended functionality. I'm leaning towards `upstream`, thoughts?

Reply:
upstream endpoint makes sense to me 🙂